.github/workflows/pythonbuild.yml

```diff
@@ -65,6 +91,10 @@ jobs:
           mysql -e 'create database IF NOT EXISTS luigi_test;' -uroot -proot || true
           mysql -e 'create user 'travis'@'localhost';' -uroot -proot || true
           mysql -e 'grant all privileges ON *.* TO 'travis'@'localhost';' -uroot -proot || true
+      - name: Create PSQL database
```

Comment:
Can you change the name of the step in line 88? it says `Setup DB` but now we have another DB, maybe we can call it Setup MySQL DB